frontend/src/app/modules/projects/project-card/project-card.component.html

```diff
@@ -1,5 +1,29 @@
-<mat-card>
-    <mat-card-content>
+<div class="project-card">
+    <div class="side-panel" [style.background-color]="accentColor"></div>
+    <div class="engine-logo-container" [style.background-color]="accentColor">
+        <img
+            *ngIf="project.engine === 'PostgreSQL'; else sqlServerLogo"
+            src="../../../../assets/postgresql.svg"
+            alt="Image Description"
+            class="engine-logo" />
+
+        <ng-template #sqlServerLogo>
+            <img src="../../../../assets/sqlserver.png" alt="Image Description" class="engine-logo" />
+        </ng-template>
```

Comment:
"/assets/postgresql.svg" will be enough. Also, it's better to implement a method in ts to get the image path for the current DbEngine, no need for ng-template here.